arch/Kconfig

```diff
@@ -269,6 +269,14 @@ config FLOAT
 	  Disabling this option means that any thread that uses a
 	  floating point register will get a fatal exception.
 
+config FP_DOUBLE_PRECISION
+	bool
+	# Omit prompt to signify "hidden" option
+	depends on FLOAT
```

Comment:
I am not sure if this should depend on FLOAT. This is more of a hardware capability.

Reply:
It depends on `FLOAT` because it is in the `FLOAT`submenu, even if the option is hidden. `FLOAT` depends on 
`CPU_HAS_FPU`, so it indirectly depends on the hardware capability.

I can modify the code so that it also depends on `CPU_HAS_FPU`.